01-docker-images/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:2.5.1
+FROM pyaillet/ruby-deps:1.0
```

Comment:
Can we use the same repo as the other images? (e.g. eu.gcr.io/kubernetes-security-workshop)

Reply:
it's uploading